Blog: welcoming conduition (#172)
bitcoinbrink/website: _posts/2026-06-01-conduition.md

cryptographic engineer with expertise in post-quantum cryptographic research and
implementation, working on the foundations that a quantum-resistant Bitcoin
might need. This is Brink's first grant for post-quantum, part of our focus on
potential threats that could affect Bitcoin's long-term security.

His work will focus on three interrelated areas: driving the [SHRINCS][shrincs]
protocol toward a draft BIP, exploring commit/reveal protocols for dormant
UTXOs, and advancing research and education around isogeny-based cryptography in
Bitcoin.

## About conduition

Conduition writes of his experience and goals:

I'm deeply honored to receive Brink's first-ever grant to focus on post-quantum
research.

This is a field I have pursued in my free time since 2024 when I published a
[survey of hash-based signature schemes][hbs survey]. In 2025 after months of
prototyping and benchmarking, I built SLHVK: [the world's fastest CPU
implementation of SLH-DSA (SPHINCS)][slhvk]. More recently, I proposed
[hypertree pruning][hypertree] - a trick which allows today's low-power hardware
wallets to generate SLH-DSA keypairs over 500x faster.

> This generous grant from Brink will help me follow through with the arc of my

Reviewer: ```suggestion
This generous grant from Brink will help me extend the arc of my
```

> previous research, as I work alongside the Blockstream team and other independent

Reviewer: ```suggestion
previous research as I work alongside the Blockstream team and other
```

> brilliant individuals to design, draft, and (hopefully) implement the first truly

Reviewer: ```suggestion
brilliant independent researches to design, draft, and hopefully implement the first truly
```

well-researched hash-based signature upgrade proposal for Bitcoin, built on the
foundations of BIP-360 and [the SHRINCS scheme][shrincs].

I also hope to use this funding to formalize commit/reveal rescue protocols, which
may someday be needed to authenticate inactive UTXOs in the presence of a quantum

> computer. Today these are folklore techniques buried in the mailing list archives,

Reviewer: ```suggestion
computer. Today, these techniques exist largely as folklore buried in mailing-list archives and are
```

in need of careful assessment and security proofs.

> Finally, I would like to expand [my prior research on isogeny cryptography][isogenies],

Reviewer: ```suggestion
Finally, I would like to expand upon [my prior research on isogeny cryptography][isogenies],
```

> which I believe has the promise to act as a suitable long-term replacement for

Reviewer: ```suggestion
which I believe holds as a suitable long-term replacement for
```

> Bitcoin's classical elliptic curve cryptography. This will take the shape of

Reviewer: ```suggestion
Bitcoin's classical elliptic-curve cryptography. This work will take the shape of
```
